student-store-ui/src/components/ProductGrid/ProductGrid.jsx

```diff
@@ -0,0 +1,97 @@
+      <form method="get" onChange={(event) => handleSearched(event)}>
+        <div className="tb">
+          <div className="td">
+              <input className="search-text" type="text" required/>
+          </div>
+          <div className="td" id="s-cover">
+            <button className="search" type="submit">
+              <div id="s-circle"></div>
+              <span></span>
+            </button>
+          </div>
+        </div>
+      </form>
+    </div>
+    {showMenu == true
+      ?<div className="menu">
+      <ul className="category-menu open">
+      <button class="toggle-menu" onClick={() => handleShowMenu()}>
+        <i className="material-icons">menu</i>
+      </button>
+          <li>
+              <button className="menu-btn" onClick={() => handleClick("")}>All Categories</button>
+          </li>
+          <li>
+              <button className="menu-btn" onClick={() => handleClick("clothing")}>Clothing</button>
+          </li>
+          <li>
+              <button className="menu-btn" onClick={() => handleClick("food")}>Food</button>
+          </li>
+          <li>
+              <button className="menu-btn" onClick={() => handleClick("accessories")}>Accessories</button>
+          </li>
+          <li>
+              <button className="menu-btn" onClick={() => handleClick("tech")}>Tech</button>
+          </li>
+      </ul>
+  </div>
+  : <button class="toggle-menu" onClick={() => handleShowMenu()}>
+        <i className="material-icons">menu</i>
+      </button>
+    }
+       
+      <div className="products-grid" id="Buy">
+        { searched != "" && clickedCategory != ""
+        ? currSearch
+          .filter((products, idx) => products.category == clickedCategory)
+          .map((product, idx) => (
+            <ProductCard  product={product} productId={product.id} 
+            quantity={product.quantity} handleAddItemsToCart ={handleAddItemsToCart} handleRemoveItemToCart={handleRemoveItemToCart} showDescription={false} key={idx}/>
+          ))
+        : searched == "" && clickedCategory == ""
+        ? products.map((product, idx) => (
+            <ProductCard  product={product} productId={product.id} 
+            quantity={product.quantity} handleAddItemsToCart ={handleAddItemsToCart} handleRemoveItemToCart={handleRemoveItemToCart} showDescription={false} key={idx}/>
+          ))
+        : searched == ""
+        ? currProds.map((product, idx) => (
+          <ProductCard  product={product} productId={product.id} 
+            quantity={product.quantity} handleAddItemsToCart ={handleAddItemsToCart} handleRemoveItemToCart={handleRemoveItemToCart} showDescription={false} key={idx}/>
+        ))
+        : currSearch.map((product, idx) =>(
+          <ProductCard  product={product} productId={product.id} 
+            quantity={product.quantity} handleAddItemsToCart ={handleAddItemsToCart} handleRemoveItemToCart={handleRemoveItemToCart} showDescription={false} key={idx}/>
+        ))
+        }
```

Comment:
nested ternarys = hard to read / prone to errors ! -> would try to clean up the conditionals to be more readable

could also do something like make a container 
```
<ProductCardContainer items={x}>
```
 which renders the cards
 ```
 <ProductCard />
 ```
 
 based on what source is passed in